content/docs/reference/api.md

```diff
@@ -241,9 +241,9 @@ _Appears in:_
 | `enabled` _boolean_ | Whether to enable the extension. |  |  |
 | `logLevel` _string_ | Log level for the agentgateway. Defaults to info.<br />Levels include "trace", "debug", "info", "error", "warn". See: https://docs.rs/tracing/latest/tracing/struct.Level.html |  |  |
 | `image` _[Image](#image)_ | The agentgateway container image. See<br />https://kubernetes.io/docs/concepts/containers/images<br />for details.<br /><br />Default values, which may be overridden individually:<br /><br />	registry: ghcr.io/agentgateway<br />	repository: agentgateway<br />	tag: <agentgateway version><br />	pullPolicy: IfNotPresent |  |  |
-| `securityContext` _[SecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v/#securitycontext-v1-core)_ | The security context for this container. See<br />https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#securitycontext-v1-core<br />for details. |  |  |
```

Comment:
These ones probably need an automation update, assuming this page is generated.

Reply:
yes, it is auto-generated, I will have to find the source in the code.

Reply:
I opened this PR to address it: https://github.com/kgateway-dev/kgateway.dev/pull/306